CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.5 FATAL_ERROR)
 
 # ---[ Project and semantic versioning.
-PROJECT(cpuinfo C CXX)
+PROJECT(
+    cpuinfo
+    LANGUAGES C CXX
+    HOMEPAGE_URL https://github.com/pytorch/cpuinfo
+    VERSION 0.1.0)
```

Comment:
cpuinfo is not versioned

Reply:
Thanks @Maratyszcza. I saw that. Do you want me to leave out `VERSION`, specify 0.0.0, or some other version? The reason why I put it in there is `pkg-config` can use the project version to determine whether the user's requested version is available. Right now [downstream](https://tracker.debian.org/pkg/cpuinfo) is manually specifying 0.0 for Debian with the git snapshot date.

Reply:
IMO best to leave out altogether. Otherwise users might have expectations that some versions are more stable than others, which isn't true as no one is doing release engineering for cpuinfo.

Reply:
Ok, I can remove it. Just keep in mind that regardless of whether you version it or not, there's over a hundred operating systems (at least Debian based) that likely have cpuinfo packaged and provide an _ad hoc_ version already. Mo Zhou of the Debian Deep Learning Team versions it in the package changelog as `0.0~git20200612.63b2545-2`.